docs/rules/unreachable-statement.md

```diff
@@ -0,0 +1,11 @@
+# unreachable-statement (C191)
+This rule is unstable and in [preview](../preview.md). The `--preview` flag is required for use.
+
+This rule is turned on by default.
+
+## What it does
+Checks for `return`, `exit`, `cycle`, and `stop` statements that result in
+unreachable code.
+
+## Why is this bad?
+Unreachable code can never be executed, and is almost certainly a mistake.
```

Comment:
One piece of feedback we had from BttFF was more examples -- could we add one here?